Subscribie/subscribie
# Qa 293 stripe subscriptions

- #293 removal of GoCardless and replace with Stripe for subscriptions

- Also introduces playwright browser automation tests (currently 4 browser tests) https://github.com/Subscribie/subscribie/blob/c13ddf80d441d40162a9ea09da1e7061d86d5263/tests/browser-automated-tests-playwright/index.js#L41

## How to QA this (manually)

Visit 
http://chrisjsimpson-qa-293-stripe-subscriptions.pcpink.co.uk/

And login as a shop owner using test credentials: 
Username: admin@example.com
Password: 
password

- Complete the 'connect to stripe' flow, using your email as a test account (you may skip sms confirmation in test mode).
- Complete the sign-up to a plan, as a customer. 
- Verify Transaction appears, new subscriber, and an invoice is present
- Complete the sign-up of a different plan

- See tests for an an idea of different payment scenarios: https://github.com/Subscribie/subscribie/blob/293-stripe-subscriptions/tests/browser-automated-tests-playwright/index.js#L41
